frontend/src/locales/de.json

```diff
@@ -316,7 +316,9 @@
           "dayOverview": "Tagesübersicht",
           "periods": "Lagerabschnitt(e)"
         },
-        "storyConfig": {
+        "summaryConfig": {
+          "contentType": "Blockinhalt",
+          "instanceNameFilter": "Instanznamenfilter",
```

Comment:
Can we find a more user-friendly name for this? E.g. "Nach Titel filtern" or "Nur mit Titel..."

Reply:
I think we rather need to rethink our naming anyway. I've created #5768

Reply:
I agree with your new issue, but I don't recall us using "Instanzname" anywhere user-facing so far. So changing this to a more understandable label wouldn't introduce more inconsistencies. Do you disagree?

Reply:
I'm not sure "Titel" is better, as it could imply that this is actually the activity title. That's why I would rather focus on this in a new issue

Reply:
Then how about Überschrift, Bezeichnung, Name, Bausteintitel, Layouttitel, ...? There is so much more background knowledge required to make sense of "Instanzname" than either of these labels. The knowledge required to understand "instanceName" is even advanced for ecamp devs, let alone nontechnical users. From the user's perspective, a content node is not an instance of anything.
I'm glad to improve it again in the follow-up issue, but I'd like to avoid shipping a completely confusing name in the first place.

Reply:
In other words, I can explain to confused users what "Nach Bausteintitel filtern" does, but I can't reasonably explain what "Instanznamenfilter" does (and make them understand why it's named that way).

Reply:
Fixed in 84bfa74